project/project/manual-repo.nix

```diff
@@ -1,26 +0,0 @@
-# this file must still be generated manually.  
```

Comment:
Can you describe changes around nix files? Those were generated by: https://github.com/input-output-hk/mantis#regenerate-lock-files?

Reply:
The only change I made to them - set the scalaVersion to 2.13.4. 
The patches from CI are applied as soon as Nix builds fail there.